frontend/src/Components/HomePage/UpcomingTrips/TravelChecklist.jsx

```diff
@@ -0,0 +1,210 @@
+import { useState } from 'react';
+import { FaPlus, FaTrash, FaChevronRight, FaChevronDown } from 'react-icons/fa';
+import PropTypes from 'prop-types';
+import './TravelChecklist.css';
+
+function useSessionStorage(key, initialValue) {
+    const [storedValue, setStoredValue] = useState(() => {
+        try {
+        const item = sessionStorage.getItem(key);
```

Comment:
qq why did you decide to use sessionStorage instead of localStorage? localStorage will persist after closing and opening your browser.